lib/provisioner/worker/plugins/automators/docker_automator/docker_automator.rb

```diff
@@ -108,18 +114,15 @@ def portmap
 
   def envmap
     # TODO: allow commas inside quotes
-    @fields['environment_variables'].split(',').map {|x| "-e #{x}" }.join(' ')
+    @fields['environment_variables'].split(',').map {|x| "-e #{x}" }.join(' ') if @fields.key?('environment_variables')
```

Comment:
It's not really related to this PR, but I notice the existing fields are all initialized in ``parse_inputmap``.  For consistency, ``@environment_variables`` (and other new fields) should be initialized to an empty array and then this check isn't needed.  but we can fix in a subsequent PR.